jruby/pom.xml

```diff
@@ -45,7 +46,6 @@
     <dependency>
       <groupId>org.jvnet.hudson</groupId>
       <artifactId>commons-jelly-tags-define</artifactId>
-      <version>1.0.1-hudson-20071021</version>
```

Comment:
Version number now defined in `<dependencyManagement>` of `stapler-parent` to reduce duplication.